apps/storybook-react/stories/Introduction.mdx

```diff
@@ -1,5 +1,4 @@
-import { Meta, Markdown } from '@storybook/blocks';
```

Comment:
Migrating to '@storybook/addon-docs/blocks' for all MDX files according to docs:  https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#consolidate-storybookblocks-into-addon-docs